src/connector/src/sink/iceberg/mod.rs

```diff
@@ -311,9 +318,64 @@ impl IcebergSink {
                 }
             };
 
+            let partition_fields = match &self.config.partition_by {
+                Some(partition_field) => {
+                    let mut partition_fields = Vec::<UnboundPartitionField>::new();
+                    // captures column, transform(column), transform(n,column), transform(n, column)
```

Comment:
Is this should be `captures column, transform(column), transform(n,column)`?

Reply:
the regex can capture the whitespace after the comma in `transform(n, column)` because of `(?:,|(,\s)))?`